### PR TITLE
adding zenodo json file for author names

### PR DESCRIPTION
**Pull request by mts299**
Hey everyone, here is the draft for zenodo authorship for pyDARN release 1.2 Hopefully soon. 

I based this on who has done the following: 
- tested
- submitted ideas 
- developed 
- reviewed code 

In the future, we may want to discuss how to ensure we aren't bloating the list if people stop helping out. 

I would also like to keep the publishing standard of the top 2 developers to get author/co-author and the rest are alphabetic. These top developers may be expanded as it is hard to know who committed the most :p

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "orcid": "0000-0002-4621-3453"
+        },
+        {
+            "affiliation": "University of Bergen",
+            "name": "Reistad, J.P.",
+            "orcid": "0000-0003-3509-5479" 
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Roberston, C."
```
Comment (Cooper10) [inline .zenodo.json:64]: '''comment
Robertson, C.R
'''

Comment (Cooper10) [inline .zenodo.json:64]:
```suggestion
            "name": "Roberston, C.R"
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "name": "Schmidt, M.T.",
+            "orcid": "0000-0002-8905-8609"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Billett, D.D.",
+            "orcid": "0000-0002-8905-8609"
+        },
+        {
+            "affiliation": "The Unviersity Centre in Svalbard",
```
Comment (carleyjmartin) [inline .zenodo.json:17]:
```suggestion
            "affiliation": "The University Centre in Svalbard",
```

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
+{
+    "creators":[
+        {
+            "name": "SuperDARN Data Analysis Working Group"
+        },
+        {
+            "affilliation": "University of Saskatchewan",
+            "name": "Schmidt, M.T.",
+            "orcid": "0000-0002-8905-8609"
```
Comment (carleyjmartin) [inline .zenodo.json:9]: This is Dan's orcid, lol.

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            "name": "Billett, D.D.",
+            "orcid": "0000-0002-8905-8609"
+        },
+        {
+            "affiliation": "The Unviersity Centre in Svalbard",
+            "name": "Bland, E.C.",
+            "orcid": "0000-0002-0252-0400"
+        },
+        {
+            "affiliation": "Naval Research Laboratory",
```
Comment (carleyjmartin) [inline .zenodo.json:22]:
```suggestion
            "affiliation": "US Naval Research Laboratory",
```

File: .zenodo.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+            "name": "Burrell, A.G.",
+            "orcid": "0000-0001-8875-9326"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Detwiller, M.H."
+        },
+        {
+            "affiliation": "The Unviersity Centre in Svalbard",
```
Comment (carleyjmartin) [inline .zenodo.json:31]:
```suggestion
            "affiliation": "The University Centre in Svalbard",
```

File: .zenodo.json [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+            "name": "Kotyk, K."
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Martin, C.J.",
+            "orcid": "0000-0002-8278-9783"
+        },
+        {
+            "affiliation": "University of Saskatchewan", 
+            "name": "Peters, D.GO."
```
Comment (carleyjmartin) [inline .zenodo.json:50]: @DannoPeters  Is GO correct?

File: .zenodo.json (hunk 3 above)
Comment (mts299) [reply .zenodo.json:9]:
```suggestion
            "orcid": "0000-0002-3265-977X"
```

File: .zenodo.json [hunk 7]
```diff
@@ -0,0 +1,75 @@
+{
+    "creators":[
+        {
+            "name": "SuperDARN Data Analysis Working Group"
+        },
+        {
+            "affilliation": "University of Saskatchewan",
```
Comment (mardet987) [inline .zenodo.json:7]:
```suggestion
            "affiliation": "University of Saskatchewan",
```

**Pull request merged**